lib/logflare/application.ex

```diff
@@ -148,6 +151,22 @@ defmodule Logflare.Application do
     goth ++ config_cat
   end
 
+  def clickhouse_conn_manager_children do
+    if SingleTenant.clickhouse_backend?() do
```

Comment:
This child spec shouldn't be needed as it should be supervised by QueryConnectionSup